superbench/common/utils/__init__.py

```diff
@@ -10,6 +10,6 @@
 nv_helper = LazyImport('superbench.common.utils.nvidia_helper')
 
 __all__ = [
-    'LazyImport', 'SuperBenchLogger', 'create_sb_output_dir', 'get_sb_config', 'logger', 'nv_helper', 'rotate_dir',
-    'network'
+    'LazyImport', 'SuperBenchLogger', 'create_sb_output_dir', 'get_sb_config', 'logger', 'network', 'nv_helper',
+    'rotate_dir'
```

Comment:
one item for one line?

Reply:
For benchmarks submodule, the items are placed in one line.

Reply:
it will exceed max line length

Reply:
Why we should concern about this rule? New line can be created if exceed the max-line-length, right? Please refer:

superbench/benchmarks/micro_benchmarks/__init__.py

Reply:
this list is updated before when it cannot be fit in one line.
isn't it more clear to have one item per line (adding a trailing comma for last item could achieve this) and sort in alphabet order if the list is too long? e.g., [when it has 50+ items](https://github.com/pytorch/pytorch/blob/v1.9.0/torch/distributions/__init__.py#L118)

Reply:
If you have these kind of concerns, we should make a rule to let all list of item to be placed in format 'one item per line', including function arguments, list, tupe and so on.
For __all__ in __init__.py, it's just to control the exporting, not for human readable.

https://github.com/python/cpython/blob/main/Lib/logging/__init__.py#L32

Reply:
different module in one project may have different styles, so nobody uses a lint rule to force `__all__`
https://github.com/python/cpython/blob/3.10/Lib/concurrent/futures/__init__.py#L20